### DISCUSSION TITLE
Mesh agent wont run

### DISCUSSION THREAD

**Discussion by T3chie-404** [Ylianst/MeshCentral#6632] [Issues]
I have Mesh Central running on a server in my data center rack and use it to manage the office computers from home. I'm attempting to add 3 new Dell pcs and download the 64bit agent through the Mesh Central server. The .exe file downloads but running it only flashes up a black cmd prompt then closes and nothing happens. What would cause the agent installer not to run? TYIA

**Comment by T3chie-404** [comment]
Also, when I run it through cmd, it gives the error:
```[2024-12-28 05:22:48 PM] [FA2A33D20580FF2A] ..\microscript\ILibDuktape_ScriptContainer.c:1667 (0,0) uncaught: 'cannot read property \x27split\x27 of null'```

**Reply by si458** [reply]
this is because you dont have wmic installed,
you need to install the wmic feature in windows 11
OR
use the new `test_agents` instead
this is done by copying them from `agents/test_agents/` to `agents/`
then restarting meshcentral and re-downloading meshagent and try again
